README.md

```diff
@@ -80,19 +80,24 @@ vars:
 ## Step 4: Enabling/Disabling Models
 Your Iterable connector might not sync every table that this package expects. If your syncs exclude certain tables, it is either because you do not use that functionality in Iterable or have actively excluded some tables from your syncs. In order to enable or disable the relevant tables in the package, you will need to add the following variable(s) to your `dbt_project.yml` file.
 
-By default, all variables are assumed to be `true` (with exception of `iterable__using_user_device_history`, which is set to `false`). 
+By default, all variables are assumed to be `true`. 
 
 
 ```yml
 vars:
     iterable__using_campaign_label_history: false                    # default is true
     iterable__using_user_unsubscribed_message_type_history: false    # default is true
     iterable__using_campaign_suppression_list_history: false         # default is true   
-    
-    iterable__using_user_device_history: true                        # default is FALSE
 ```
 
-## (Optional) Step 5: Additional configurations
+## Step 5: Unsubscribe tables are no longer history tables
```

Comment:
Same comment from the source, this shouldn't be it's own step and should rather be a component of the pre-req section.

Reply:
updated